cli/toolchain_cmd.go

```diff
@@ -605,7 +611,7 @@ func cloneToolchain(dest, toolchain string) error {
 			return err
 		}
 
-		cmd := exec.Command("git", "clone", "https://"+toolchain)
+		cmd := exec.Command("git", "clone", toolchain)
```

Comment:
Please rename `toolchain` to `cloneURL` or `cloneURI`. The func is already called `cloneToolchain`, so it's clear that we're dealing with toolchains, but `toolchain` variable is very hard to understand what it actually is.